jupyterlab/commands.py

```diff
@@ -34,7 +34,10 @@ def run(cmd, **kwargs):
     print('> ' + list2cmdline(cmd))
     kwargs.setdefault('shell', sys.platform == 'win32')
     kwargs.setdefault('env', os.environ)
-    return check_output(cmd, **kwargs)
+    try:
+        return check_output(cmd, **kwargs)
+    except CalledProcessError as error:
+        raise error
```

Comment:
Sorry, not a Python native here. I initially `print`ed the error but that wasn't helpful because it wasn't formatted so I naively put a `raise` in here instead. For some reason, putting `check_output` in a `try` block resolves this issue of webpack errors not being passed through to the parent process. So, we gotta put something in the `except` block, right?